composer.json

```diff
@@ -17,13 +17,13 @@
         "php": "^7.2||^8.0",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
         "doctrine/coding-standard": "^6.0",
-        "squizlabs/php_codesniffer": "^3.4.0"
+        "squizlabs/php_codesniffer": "^3.5"
     },
     "require-dev": {
         "jakub-onderka/php-parallel-lint": "^1.0",
-        "phpstan/phpstan": "^0.11.7",
-        "phpstan/phpstan-phpunit": "^0.11.2",
-        "phpstan/phpstan-strict-rules": "^0.11.0",
+        "phpstan/phpstan": "^0.11",
```

Comment:
we should probably lock to patch version, as phpstan's maintainer doesn't consider patch version to be patch version. When new "patch" version is released and new violation is found it breaks the build.

Reply:
oh... ok...sucks

Reply:
There was [ackward discussion](https://github.com/phpstan/phpstan/issues/2163) about it.